### PR TITLE
OX-4086 | Add extended snippet to sarif code flows

### PR DESCRIPTION
**Pull request by oxeye-mher**
[OX-4086](https://oxeye.atlassian.net/browse/OX-4086?atlOrigin=eyJpIjoiY2U5YWZjZWNhMmE4NDk0MGE1YjM0MmNkOThlYTc5NDgiLCJwIjoiaiJ9)

[OX-4086]: https://oxeye.atlassian.net/browse/OX-4086?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: cli/src/semgrep/formatter/sarif.py [hunk 1]
```diff
@@ -30,17 +53,22 @@ def _create_sarif_location_dict(
...
+            end_highlight,
+        ) = SarifFormatter._generate_taint_snippet(location)
         sarif_dict = {
             "location": {
                 "message": {"text": message},
                 "physicalLocation": {
                     "artifactLocation": {"uri": str(rule_match.path)},
                     "region": {
-                        "startLine": location.start.line,
+                        "startLine": start_highlight,
```
Comment (oxeye-meir) [inline cli/src/semgrep/formatter/sarif.py]: `startLine`??
or you meant `startColumn`?

Comment (oxeye-mher) [reply cli/src/semgrep/formatter/sarif.py]: you're correct

Comment (oxeye-mher) [reply cli/src/semgrep/formatter/sarif.py]: done

**Review by oxeye-meir — APPROVED** [review]
LGTM
Just fix the linter issues

**Pull request merged**